src/main/java/com/redhat/devtools/lsp4ij/client/features/LSPImplementationFeature.java

```diff
@@ -0,0 +1,39 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Red Hat, Inc.
+ * Distributed under license by Red Hat, Inc. All rights reserved.
+ * This program is made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution,
+ * and is available at https://www.eclipse.org/legal/epl-v20.html
+ *
+ * Contributors:
+ * Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package com.redhat.devtools.lsp4ij.client.features;
+
+import com.intellij.psi.PsiFile;
+import com.redhat.devtools.lsp4ij.LanguageServerItem;
+import org.jetbrains.annotations.ApiStatus;
+import org.jetbrains.annotations.NotNull;
+
+/**
+ * LSP implementation feature.

```

Comment:
There's no such thing as LSP implementation feature. Is it Go to Implementation?

Reply:
Yes with menu

Reply:
See https://github.com/redhat-developer/lsp4ij/blob/main/docs/LSPSupport.md#implementation